api/src/main/java/org/apache/iceberg/io/ParallelIterable.java

```diff
@@ -28,25 +28,25 @@
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.Future;
 import org.apache.iceberg.exceptions.RuntimeIOException;
-import org.apache.iceberg.io.CloseableGroup;
-import org.apache.iceberg.io.CloseableIterable;
-import org.apache.iceberg.io.CloseableIterator;
 import org.apache.iceberg.relocated.com.google.common.base.Preconditions;
 import org.apache.iceberg.relocated.com.google.common.collect.Iterables;
 
 public class ParallelIterable<T> extends CloseableGroup implements CloseableIterable<T> {
```

Comment:
Moving this class is a breaking change. I'd be okay with it if the purpose was to make it private (and there was an appropriate deprecation in its place). But since this isn't private, I'm not sure the purpose or value of moving it.